src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,2 @@
+package core.basesyntax;public class Rectangle {
```

Comment:
The class declaration should not begin with an empty line. Please remove the empty line at the beginning of the class implementation.